bin/cli.js

```diff
@@ -30,8 +33,10 @@ const queryGenerator = new QueryGenerator(serverUrl);
 
 let failedTests = 0;
 let passedTests = 0;
+let retryCount = program.retry || 0;
+let timeout = program.timeout || 0;
```

Comment:
not sure if `0` is a good default value maybe `1000` or something would be better?